lisp/ess-julia.el

```diff
@@ -389,7 +436,8 @@ It makes underscores and dots word constituent chars.")
   (add-hook 'completion-at-point-functions 'ess-julia-object-completion nil 'local)
   (add-hook 'completion-at-point-functions 'ess-filename-completion nil 'local)
   (if (fboundp 'ess-add-toolbar) (ess-add-toolbar))
-  (set (make-local-variable 'end-of-defun-function) 'ess-end-of-function)
+  (setq-local beginning-of-defun-function #'ess-beginning-of-function)
```

Comment:
I though about adding this too, but `ess-beginning-of-function` doesn't have the same semantics as `beginning-of-defun`, although maybe that isn't an issue.